utils/format-modules.js

```diff
@@ -17,7 +17,7 @@ function _formatFileName(mod) {
 
   // Source file.
   if (!baseName) {
-    return `{green-fg}./${relative(process.cwd(), resolve(fileName))}{/}`;
+    return `{green-fg}.${sep}${relative(process.cwd(), resolve(fileName))}{/}`;
```

Comment:
Ugh. This makes this PR a (minor, minor, ) minor semver release....

Reply:
No worries, I'll take care of it after merge!